whitepaper/chapters/system.tex

```diff
@@ -180,8 +180,8 @@ \subsection{Broker}
 There is no way for the broker to send messages to the server.
 This decoupling is intentional and was done for performance reasons.
 
-The server raises subscription events in the block chain sync consumer \nemrefparens{sec:disruptor:blockConsumers} when it holds an exclusive lock to the blockchain data.
-In order to prevent slow database operations from occurring when the server has an exclusive lock, these operations are offloaded to the broker.
+The server raises subscription events in the blockchain sync consumer \nemrefparens{sec:disruptor:blockConsumers} when it holds an exclusive lock to the blockchain data.
+To prevent slow database operations when the server has an exclusive lock, these operations are offloaded to the broker.
```

Comment:
i would reverse phrases

Reply:
Do you mean:

A) Change line 183 for 184 or change 184.
B) Change 184 to ``These operations are offloaded to the broker to prevent slow database operations when the server has an exclusive lock.``

Reply:
B